cpp/example_code/glue/glue_getting_started_scenario.cpp

```diff
@@ -0,0 +1,889 @@
+         \param bucketName: An Amazon S3 bucket created in the setup.
+         \param filePath: The path of the file to upload.
+         \param fileName The name for the uploaded file.
+         \param clientConfig Aws client configuration.
+         \return bool: Successful completion.
+         */
+        static bool uploadFile(const Aws::String &bucketName,
+                               const Aws::String &filePath,
+                               const Aws::String &fileName,
+                               const Aws::Client::ClientConfiguration &clientConfig);
+
+        //! Routine which deletes all objects in an Amazon S3 bucket.
+        /*!
+         \\sa deleteAllObjectsInS3Bucket()
+         \param bucketName: The Amazon S3 bucket name.
+         \param clientConfig Aws client configuration.
+         \return bool: Successful completion.
+         */
+        static bool deleteAllObjectsInS3Bucket(const Aws::String &bucketName,
+                                               const Aws::Client::ClientConfiguration &clientConfig);
+
+        //! Routine which retrieves an object from an Amazon S3 bucket.
+        /*!
+         \\sa getObjectFromBucket()
+         \param bucketName: The Amazon S3 bucket name.
+         \param objectKey: The object's name.
+         \param objectStream: A stream to receive the retrieved data.
+         \param clientConfig Aws client configuration.
+         \return bool: Successful completion.
+         */
+        static bool getObjectFromBucket(const Aws::String &bucketName,
+                                        const Aws::String &objectKey,
+                                        std::ostream &objectStream,
+                                        const Aws::Client::ClientConfiguration &clientConfig);
+
+
+        //! Cleanup routine to delete created assets.
+        /*!
+         \\sa deleteAssets()
+         \param crawler: Name of an AWS Glue crawler.
+         \param database: The name of an AWS Glue database.
+         \param job: The name of an AWS Glue job.
+         \param bucketName: The name of an Amazon S3 bucket.
+         \param clientConfig Aws client configuration.
+         \return bool: Successful completion.
+         */
+        static bool
+        deleteAssets(const Aws::String &crawler,
+                     const Aws::String &database,
+                     const Aws::String &job,
+                     const Aws::String &bucketName,
+                     const Aws::Client::ClientConfiguration &clientConfig);
+    } // Glue
+} // AwsDoc
+
+// snippet-start:[cpp.example_code.glue.glue_getting_started_scenario]
+//! Scenario which demonstrates using AWS Glue to add a crawler and run a job.
+/*!
+ \\sa runGettingStartedWithGlueScenario()
+ \param bucketName: An Amazon Simple Storage Service (Amazon S3) bucket created in the setup.
+ \param roleName: An AWS Identity and Access Management (IAM) role created in the setup.
+ \param clientConfig Aws client configuration.
```

Comment:
Capitalize AWS.

Reply:
Unsure about capitalizing AWS in this scenario. Does it refer to the "Aws" const in the following code?

Reply:
I omitted the semicolon after clientConfig. It's documentation.